### DISCUSSION TITLE
Group Assignment #2-1: Data Plan

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#35] [Assignments]
Instructions [here](https://github.com/yohman/21S-DH151/blob/main/Group%20Assignments/GroupAssignment2.md).

Include:

- Group name
- Group members
- Link to data plan in your group repo

**Comment by michelle14lee** [comment]
Group name: Datart
Group members: Annmarie Cong, Eustina Kim, Michelle Lee
Data Plan: https://github.com/kimjee8955/DH151-Datart/blob/main/Data%20Plan%20Update.md

**Comment by frida-barragan** [comment]
Group name: Health mappers
Group members: Frida Barragan & Kate McInerny
Data Plan: https://github.com/frida-barragan/DH151_Group/blob/main/Data_plan.md

**Comment by ryanorlando** [comment]
Group Name: Team Data
Group Members: Ryan Orlando, Ainsleigh Douglas, and Sebastien Dills
Data Plan: https://github.com/ryanorlando/DH151-Project/blob/main/Assignment2/DataPlanUpdate.md

**Comment by alexthegreat1** [comment]
Group Name: cARTographers
Group Members: Kai Colorado & Alex Haddad
Data Plan: https://github.com/kaicolorado/DH151-Group/blob/main/Group%20Assignments/Data%20Plan.md

**Comment by danielliuzy** [comment]
Group Name: The Musical Maps
Group Members: Justis Guardia, Daniel Liu, Kai Watanabe
Data Plan: https://github.com/justisg/TheMusicalMaps/blob/main/dataplan.md

**Comment by zonaliao** [comment]
Group Name: State-of-the-Climate
Group Members: Alekhya Vittalam, Natasha Grace, Zona Liao
Data Plan: https://github.com/alekhyavittalam/state-of-the-arts/blob/main/Week4/dataplan.md

**Comment by lisa58** [comment]
Group Name: Fantastic Four
Group Members: Labiba Alam, Haiqi Zhou, Adriana Romero, and Lynn Tieu
Data Plan: https://github.com/LynnT2/FantasticFour/blob/main/dataplanupdate.md

**Comment by jmtanenbaum** [comment]
Group Name: Environmental Racism in Carceral California 
Group members: Julia Tanenbaum, Lisa Kahn, Hannah Rogers https://github.com/jmtanenbaum/DHEnviroIncarceration/blob/main/Updated%20Data%20Plan.md

**Comment by derekwang99** [comment]
Group Name: Ocean Pollution Posse
Group Members: Julia Wood, Eli Aranita, Derek Wang
https://github.com/eliaranita1/project1/blob/main/Group%20Assignments/%232/data_plan_update.md
